packages/use-insertion-effect-with-fallbacks/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@emotion/use-insertion-effect-with-fallbacks",
```

Comment:
Should this be `*-with-fallback` or `*-with-fallbacks`? :sweat_smile: #realproblems